src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -53,41 +61,13 @@
   public static final String RETRIEVE_IRM_URL_TEMPLATE = "/dataset/{datasetId}/irm";
   public static final MediaType IRM_MEDIA_TYPE = new MediaType("application", "dni-tdf+xml");
   public static final String IRM_MEDIA_TYPE_VALUE = "application/dni-tdf+xml";
-
+  public static final MediaType METACARD_MEDIA_TYPE = MediaType.APPLICATION_XML;
   @NotNull private final StoreService storeService;
   @NotBlank private final String storeApiVersion;
 
-  /**
-   * TODO Use {@link org.springframework.web.server.ResponseStatusException} instead of catching
-   * {@link Exception}s
-   */
   @Override
-  public ResponseEntity<Void> createDataset(
-      final String acceptVersion, @Valid final MultipartFile file) {
-    final String expectedAcceptVersion = storeApiVersion;
-    if (!StringUtils.equals(acceptVersion, expectedAcceptVersion)) {
-      throw new UnsupportedOperationException(
-          String.format(
-              "%s was \"%s\", but only \"%s\" is currently supported.",
-              ACCEPT_VERSION_HEADER_NAME, acceptVersion, expectedAcceptVersion));
-    }
-
-    MultipartFileValidator.validate(file);
-    final String mediaType = file.getContentType();
-    final String fileName = file.getOriginalFilename();
-
-    final URI location;
-    try (final InputStream inputStream = file.getInputStream()) {
-      location = storeService.createDataset(file.getSize(), mediaType, fileName, inputStream);
-    } catch (IOException e) {
-      throw new ValidationException(
-          String.format(
-              "Unable to read file for createDataset request with mediaType=%s and fileName=%s",
-              mediaType, fileName),
-          e);
-    }
-
-    return ResponseEntity.created(location).build();
+  public Optional<NativeWebRequest> getRequest() {
```

Comment:
Delete this method

Reply:
If we want to implement both APIs we have to overwrite this method since they both have it as a default.

Reply:
This will go away with my changes because I am removing OpenAPI spec.